allenact_plugins/manipulathor_plugin/manipulathor_constants.py

```diff
@@ -0,0 +1,116 @@
+"""Constant values and hyperparameters that are used by the environment."""
+import ai2thor
+import ai2thor.fifo_server
+from allenact_plugins.ithor_plugin.ithor_environment import IThorEnvironment
+
+MOVE_THR = 0.01
+ARM_MIN_HEIGHT = 0.450998873
+ARM_MAX_HEIGHT = 1.8009994
+MOVE_ARM_CONSTANT = 0.05
+MOVE_ARM_HEIGHT_CONSTANT = MOVE_ARM_CONSTANT
```

Comment:
The `MOVE_ARM_CONSTANT` and `MOVE_ARM_HEIGHT_CONSTANT` constants seem to be `ArmPointNav` specific? Might it make more sense to have these be changeable defaults? See, for instance, how the `VISIBILITY_DISTANCE` constant from the `ithor_plugin` is used.